universal-application-tool-0.0.1/app/services/question/types/MultiOptionQuestionDefinition.java

```diff
@@ -155,6 +155,14 @@ protected MultiOptionQuestionDefinition(
     }
   }
 
+  /** Get the default locale representation of the option with the given ID. */
+  public Optional<String> getDefaultLocaleOptionForId(long id) {
```

Comment:
always having to hone these services :D It's sweet that it was designed in such a way that you could just add this method!